apps/web/src/serlo-editor-integration/create-plugins.tsx

```diff
@@ -189,16 +193,34 @@ export function createPlugins({
       plugin: exercisePlugin,
       visibleInSuggestions: true,
     },
-    { type: EditorPluginType.Solution, plugin: solutionPlugin },
-    { type: EditorPluginType.H5p, plugin: H5pPlugin },
+    {
+      type: EditorPluginType.Solution,
+      plugin: solutionPlugin,
+      visibleInSuggestions: true,
+    },
+    {
+      type: EditorPluginType.H5p,
+      plugin: H5pPlugin,
+      visibleInSuggestions: true,
+      icon: <IconH5p />,
+    },
     {
       type: EditorPluginType.InputExercise,
+      visibleInSuggestions: true,
       plugin: createInputExercisePlugin(),
+      icon: <IconTextArea />,
+    },
+    {
+      type: EditorPluginType.ScMcExercise,
+      visibleInSuggestions: true,
+      plugin: createScMcExercisePlugin(),
+      icon: <IconScMcExercise />,
```

Comment:
Super minor: It would be super neat to keep the property order consistent across all the plugins. It makes it easier to scan the list and see what sticks out, the brain (mine, at least) is way faster if it can expect a certain prop in a certain position. Again, minor.

Reply:
Fixed